config/locales/en.yml

```
@@ -2977,11 +2980,13 @@ en:
           sales_tax_content: This is a tax collected at the point of sale when you buy goods within your state. However, if you purchased items through an out-of-state seller, they may not have collected Idaho sales tax at the time.
           sales_tax_title: What are sales taxes?
           sales_use_tax_helper_text: Enter total amount
-          subtitle_html: This includes online purchases where sales tax was <strong>not</strong> applied.
+          subtitle_html: This includes online purchases where sales tax was <strong>not</strong> applied. If you made any purchases without paying sales tax, you may owe use tax on those purchases. We’ll help you figure out the right amount
           title:
             one: Did you make any out-of-state purchases in %{year} without paying sales tax?
             other: Did you or your spouse make any out-of-state purchases in %{year} without paying sales tax?
-          unpaid_sales_use_tax_label_html: Please enter the <b>total amount of purchases</b> that didn’t have sales or use tax applied.
+          unpaid_sales_use_tax_label_html: Enter the <b>total amount of purchases</b> that didn’t have sales tax applied in %{year}.
```

Use strong tags instead of b tags. We usually try to avoid b tags as are listed as [WCAG compliance](https://siteimprove.com/en-us/accessibility/wcag-compliance-website-checker/) Level A Errors. Screen readers do not normally announce these tags to the user. ([source](https://help.siteimprove.com/support/solutions/articles/80000448460-accessibility-the-use-of-emphasis-in-text))
```suggestion
          unpaid_sales_use_tax_label_html: Enter the <strong>total amount of purchases</strong> that didn’t have sales tax applied in %{year}.
```